server/routes/users.js

```diff
@@ -0,0 +1,29 @@
+var express = require('express');
+var router = express.Router();
+
+const users = [
+    { username: 'admin', password: 'admin' },
+    { username: 'hams', password: 'awesome' },
+    { username: 'kevin', password: 'lawyer' }
+];
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+    res.send('respond with a resource');
+});
+
+router.post('/', function(req, res, next) {
+    const reqBody = req.body;
+
+    var count = users.length;
+
+    for (var i = 0; i < count; i++) {
+        const user = users[i];
+        if (user.username === reqBody.username && user.password === reqBody.password) {
+            res.send('good id');
```

Comment:
need the return statement, or it'll try to do another send after the for loop. you'll prob see a console error message when that happens.

```suggestion
            res.send('good id');
            return;
```